### ISSUE TITLE
auto_arima often fails to find a model with statsmodel 0.11.0

### ISSUE THREAD

**Issue by EricWrightAtWork** [alkaline-ml/pmdarima#294]
**Describe the bug**
auto_arima often fails to find a model with statsmodel 0.11 and throws the ValueError: "Could not successfully fit ARIMA to input data. It is likely your data is non-stationary. Please induce stationarity or try a different range of model order params. If your data is seasonal, check the period (m) of the data."
This is very undesirable for a general forecasting solution - in our case, we'd always prefer to fit a model, even if it's not a great one. I understand this is likely more of a statsmodels issue, but maybe you all can shed some light on the differences in the latest stable statsmodels version and if you have plans to work around this.
As an example, I have a unit test using some synthetic data that passed with older versions of statsmodels (i.e. <=0.10.2) but now always fails with 0.11.0. Main code from the test is below.  

**To Reproduce**  

```python
import numpy as np
import pmdarima

time_index = np.arange(20)
amplitude = 15.0
offset = 85.0
period = 5
signal_vals = np.sin(time_index / period * 2 * np.pi) * amplitude + offset

# Compute noise values
noise_amplitude = 3.0
noise_vals = np.random.uniform(low=-noise_amplitude, high=noise_amplitude, size=(time_index.size,))

# Combine and model
vals = signal_vals + noise_vals
pmdarima.arima.auto_arima(vals)
```

**Versions**
1.5.2

**Expected behavior**

**Actual behavior**

**Additional context**

**Comment by tgsmith61591** [comment]
Yeah good catch. I merged a fix for this the other day in #293, hoping to cut a release this week. Thanks for the report!

**Comment by goldentom42** [comment]
Thanks for the quick fix @tgsmith61591 !!!

**Comment by EricWrightAtWork** [comment]
Awesome to hear - thank you! Looking forward to the new release :)

**Comment by aaronreidsmith** [comment]
This has been fixed with the deploy of `1.5.3` this morning
